assets/data-port/import.js

```diff
@@ -23,6 +28,20 @@ const SenseiImportPage = () => {
 
 	const { fetchCurrentJobState } = useDispatch( 'sensei/import' );
 
+	const [ currentStep, setCurrentStep ] = useState();
+
+	useEffect( () => {
+		const newStep = navigationSteps.find( ( step ) => step.isNext );
+
+		if ( 'complete' === newStep.key ) {
+			setTimeout( () => {
+				setCurrentStep( newStep );
+			}, 1000 ); // CSS animation time to complete the progress bar.
+		} else {
```

Comment:
> Update the frontend to delay 1 second before moving to the Done step. So the user can see the progress bar full.

I'm not sure we have to add this artificial delay. It's fine if they don't see a 100% but jump to the results as soon as available. 
(In fact, the original problem with this progress bar was that it showed 100% for a bit before showing the results)

Reply:
I reverted that change for now. 😉 https://github.com/Automattic/sensei/pull/3485/commits/69b3a25c345568f7dc72825be07f65fedce4fbcb